java/rocksjni/cassandra_compactionfilterjni.cc

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree. An additional grant
+// of patent rights can be found in the PATENTS file in the same directory.
+
```

Comment:
We recently updated the license to Apache+GPLv2, and updated the copyright headers in all the files in RocksDB. So please use those headers here and in all the new files.